tf2_kdl/include/tf2_kdl/tf2_kdl.h

```diff
@@ -108,9 +112,9 @@ geometry_msgs::PointStamped toMsg(const tf2::Stamped<KDL::Vector>& in)
  * \param out The point converted to a timestamped KDL Vector.
  */
 inline
-void fromMsg(const geometry_msgs::PointStamped& msg, tf2::Stamped<KDL::Vector>& out)
+void fromMsg(const geometry_msgs::msg::PointStamped& msg, tf2::Stamped<KDL::Vector>& out)
 {
-  out.stamp_ = msg.header.stamp;
+  out.stamp_ = tf2::TimePoint(durationFromSec(msg.header.stamp.sec + msg.header.stamp.nanosec/NSEC_PER_SECOND));
```

Comment:
You apply here a simplified version of what's been used at https://github.com/ros2/geometry2/pull/90/files#diff-2aeab99ef52e795be884c1fbbef16e44R195 where the function used was defined at https://github.com/ros2/geometry2/pull/90/files#diff-2aeab99ef52e795be884c1fbbef16e44R78.

This needs to get repeated over a few places so IMHO, creating a function sounds more appropriate.